raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -1542,8 +1541,13 @@ def handle_node_change_network_state(
     payer_channel = channelidentifiers_to_channels.get(payer_channel_identifier)
     payee_channel = channelidentifiers_to_channels.get(route.channel_identifier)
 
+    # TODO: Is this state transition correct here? payer_channel was not checked for None
+    # but later down the call stack it's assumed to not be None.
+    if not payee_channel or not payer_channel:
```

Comment:
Don't really get this todo. Isn't what this if statement doing checking both payer and payee?

Reply:
I took the liberty to remove it because i am assuming you missed the `not payer_channel` part. Let me know if i understood you incorrectly and we can restore the todo.